libcxx/test/libcxx/containers/sequences/deque/assert.pass.cpp

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <deque>
+
+// Test hardening assertions for std::deque.
+
+// REQUIRES: has-unix-headers
```

Comment:
Nit: for completeness' sake, this also needs:
```
// UNSUPPORTED: c++03
// XFAIL: libcpp-hardening-mode=debug && availability-verbose_abort-missing
```
(due to the way `check_assertion.h` is implemented). I plan to add a new feature like `has-check-assertion` to wrap all those checks together, but for now it all has to be marked explicitly in tests relying on `check_assertion.h`.

Reply:
Done